### ISSUE TITLE
using "is" with gdscript class on both sides fail

### ISSUE THREAD

**Issue by henkz1** [godotengine/godot#11643]
**Operating system or device, Godot version, GPU Model and driver (if graphics related):**
Ubuntu 17.04, godot master da144fed4c49133a93b3b83cfc665352c0af351a

**Issue description:**
Checking if an instance that inherits a gdscript class is an instance of a gdscript class fails with ```Internal Script Error - opcode #31 (report please)```

Pretty sure this was introduced in https://github.com/godotengine/godot/pull/11518 just like https://github.com/godotengine/godot/issues/11595

**Steps to reproduce:**
```
class ClassA:
	var x
	
func _ready():
	ClassA.new() is Node # works
	Node.new() is ClassA # works
	ClassA.new() is ClassA # Internal Script Error - opcode #31
```

**Comment by Sslaxx** [comment]
It also fails on checking for classes via files.

`	if (!ring_taken && (body is preload ("res://Scripts/player.gd"))):` - fails with the same error as above.

**Comment by henkz1** [comment]
@Sslaxx checking with preloaded class is supposed to work, that's how I discovered this bug in my project.
AFAIK the difference between load and preload is that preload loads at parse time, so preload call wont block when called the first time like load would.
And yea it looks like #11596 has transformed to be a duplicate of this issue, or maybe this issue is the duplicate now..

**Comment by hpvb** [comment]
This is fixed in 6c15c23889772ce4986765e801be3f216ed50a11
